dynamodb/src/main/scala/zio/dynamodb/Codec.scala

```diff
@@ -372,6 +372,31 @@ private[dynamodb] object Codec {
 
   private[dynamodb] object Decoder extends GeneratedCaseClassDecoders {
 
+    sealed trait ContainerField
```

Comment:
I noticed a bug in decoding container fields when they are missing so (probably unwisely) decide to fix it in this PR